README.md

```diff
@@ -1 +1,173 @@
+- [📖 \[Books-store\] ](#-Books-store-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Build](#build)
+    - [Run](#run)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [© Acknowledge ](#-acknowledge-)
+  - [⭐ Show your support ](#-show-your-support-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 [Books-store] <a name="about-project"></a>
+
+*[Math Magicians]* a WebApp that shows two pages: Books and Categories. here, users can find a list of books they have added and check the reading status. They can also delete the ones they already finished.
+<div align="center">
+
+  <img src="./src/images/screenshot.png" alt="bookstore" width="750"  height="auto" />
+
+  <br/>
+
+</div>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML" target="_blank" rel="noopener noreferrer">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS" target="_blank" rel="noopener noreferrer">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript" target="_blank" rel="noopener noreferrer">Javascript</a></li>
+    <li><a href="https://webpack.js.org/" target="_blank" rel="noopener noreferrer">WebPack</a></li>
+    <li><a href="https://es.reactjs.org/" target="_blank" rel="noopener noreferrer">React</a></li>
+  </ul>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- *Dinamically displays the pages that were build by using react function based components.*
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="" target="_blank" rel="noopener noreferrer">Math magicians - Live Demo</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need `git` locally installed. Please got to [Getting Started - Installing Git guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the steps described for your system to install `git`.
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  cd my-folder
+  git clone git@github.com:juanpa8830/React-Math-magicians.git
+```
+
+### Install
+
+Install this project with:
+```sh
+  cd math magicians
+  npm install
+```
+
+### Build
+
+To build the project, execute the following command:
+```sh
+  npm run build  
+```
+
+### Run
+
+To run tests, run the following command:
+
+  npm test
+
```

Comment:
- [ ] You didn't add npm start command which is used to start the application :fearful: Kindly add this command in this section to make your readme file descriptive and professional :blush: